[2023-09-20T20:33:19.393+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: student_semantic.create_dataframe_student manual__2023-09-20T20:33:18.425082+00:00 [queued]>
[2023-09-20T20:33:19.402+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: student_semantic.create_dataframe_student manual__2023-09-20T20:33:18.425082+00:00 [queued]>
[2023-09-20T20:33:19.402+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-20T20:33:19.414+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): create_dataframe_student> on 2023-09-20 20:33:18.425082+00:00
[2023-09-20T20:33:19.420+0000] {standard_task_runner.py:57} INFO - Started process 1952 to run task
[2023-09-20T20:33:19.424+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'student_semantic', 'create_dataframe_student', 'manual__2023-09-20T20:33:18.425082+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/student_semantic.py', '--cfg-path', '/tmp/tmp6x_50bqh']
[2023-09-20T20:33:19.426+0000] {standard_task_runner.py:85} INFO - Job 200: Subtask create_dataframe_student
[2023-09-20T20:33:19.469+0000] {task_command.py:415} INFO - Running <TaskInstance: student_semantic.create_dataframe_student manual__2023-09-20T20:33:18.425082+00:00 [running]> on host 6086152831d6
[2023-09-20T20:33:19.728+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='student_semantic' AIRFLOW_CTX_TASK_ID='create_dataframe_student' AIRFLOW_CTX_EXECUTION_DATE='2023-09-20T20:33:18.425082+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-20T20:33:18.425082+00:00'
[2023-09-20T20:33:19.738+0000] {logging_mixin.py:151} INFO - json_data None
[2023-09-20T20:33:19.738+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/packages/student_semantic/student_semantic.py", line 44, in create_dataframe_student
    df = pd.json_normalize(xcom_pull_obj_student[key_to_normalize], sep="_")
TypeError: 'NoneType' object is not subscriptable
[2023-09-20T20:33:19.746+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=student_semantic, task_id=create_dataframe_student, execution_date=20230920T203318, start_date=20230920T203319, end_date=20230920T203319
[2023-09-20T20:33:19.758+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 200 for task create_dataframe_student ('NoneType' object is not subscriptable; 1952)
[2023-09-20T20:33:19.797+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-20T20:33:19.810+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
